### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by 7Bpencil**
I am trying to recreate paper results, but there are various oddities in the code, fixed some of them

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -937,7 +937,7 @@ def set_causal(self, causal):
 
     def forward(self, x, mask=None):
         # x shape: [B, Feat, Exp, S]
-        print(x.shape) #torch.Size([64, 7, 5, 96])
+        # print(x.shape) #torch.Size([64, 7, 5, 96])
```
Comment (7Bpencil) [inline models.py]: this print spams console with thousands (or hundreds of thousands) messages

File: src/callback/scheduler.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def before_fit(self):
                                             final_div_factor=self.final_div_factor,
                                             three_phase=self.three_phase,
                                             last_epoch=self.last_epoch,
-                                            verbose=self.verbose
+                                            # verbose=self.verbose
```
Comment (7Bpencil) [inline src/callback/scheduler.py]: this parameter doesn't exist, see https://docs.pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html

File: main.py [hunk 3]
```diff
@@ -392,11 +392,11 @@ def plot_feature_actual_vs_predicted(actual, predicted, feature_idx):
...
-        #suggested_lr = find_lr()
-        #print('Suggested LR for model_a:', suggested_lr)
-        #def count_parameters(model_a): # count number of trainable parameters in the model
-        #     return sum(p.numel() for p in model.parameters() if p.requires_grad)
-        #train_func(model_type='model_a', lr=suggested_lr)
+        suggested_lr = find_lr()
+        print('Suggested LR for model_a:', suggested_lr)
+        def count_parameters(model_a): # count number of trainable parameters in the model
+            return sum(p.numel() for p in model.parameters() if p.requires_grad)
+        train_func(model_type='model_a', lr=suggested_lr)
```
Comment (7Bpencil) [inline main.py]: what was the reason to comment out model_a training?
